src/commands/hearthstone.js

```diff
@@ -0,0 +1,38 @@
+const Aquarius = require('../aquarius');
+const hdb = require('hearthstone-db');
+
+class Hearthstone extends Aquarius.Command {
+  constructor() {
+    super();
+    this.description = 'Displays Hearthstone Card Image';
+  }
+
+  helpMessage() {
+    let msg = super.helpMessage();
+
+    msg += 'Usage:\n';
+    msg += '```[[Tirion Fordring]]```';
+    return msg;
+  }
+
+  message(msg) {
+    const hearthInput = Aquarius.Triggers.cardTrigger(msg);
+
+    if (hearthInput) {
+      hearthInput.forEach(cardInput => {
+        let cardName = `${cardInput[1]}`;
+        this.log(`Request for ${cardName}`);
+
+        Aquarius.Loading.startLoading(msg.channel);
+        hdb.allCards.forEach(card => {
+          if (card.name.toLowerCase() === cardName.toLowerCase()) {
+            return msg.channel.sendFile(card.image_url, `${card.name}.png`);
```

Comment:
Ah, I see what you're doing. Ok, so [.forEach(...)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) isn't the method you want probably. My guess is you'll want to change it to [.some(...)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) and then remove the `return` from this line and add a new line right below that just `return true;`.